netscaler/config.go

```diff
@@ -463,6 +463,32 @@ func (c *NitroClient) EnableFeatures(featureNames []string) error {
 	return nil
 }
 
+func (c *NitroClient) DisableFeatures(featureNames []string) error {
+	/* construct this:
+	{
+	        "nsfeature":
+		{
+		    "feature": [ "LB", ]
+		}
+	}
+	*/
+	featureStruct := make(map[string]map[string][]string)
+	featureStruct["nsfeature"] = make(map[string][]string)
+	featureStruct["nsfeature"]["feature"] = featureNames
+
+	featureJSON, err := JSONMarshal(featureStruct)
+	if err != nil {
+		log.Printf("[ERROR] go-nitro: EnableFeatures: Failed to marshal features to JSON")
```

Comment:
Log message should mention 'DisableFeatures'

Reply:
Fixed with 05766c1